src/Microsoft.Health.Utility/Microsoft.Health.Utility.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Code itself looks good and I like the update to use a delegate to make the function more generic.  My question is do we need a new library or can this be part of an existing one like https://github.com/microsoft/healthcare-shared-components/tree/main/src/Microsoft.Health.Core?  It seems like a reasonable fit there.  If we keep this library, what is the guidance for when new code would go here vs other libraries?

Reply:
> The utility tool bag can be consumed by other packages too with clear meaning

I'd ague the opposite 😄 by definition, a general purpose utility library's actual purpose can't be well defined

Reply:
> > The utility tool bag can be consumed by other packages too with clear meaning
> 
> I'd argue the opposite 😄 by definition, a general purpose utility library's actual purpose can't be well defined

That sounds true. :) The other option I can think of is to rename it utility.measurements.